build-system/app.js

```diff
@@ -1199,4 +1198,4 @@ function generateInfo(filePath) {
       '<h3><a href = /serve_mode=cdn>Change to CDN mode (prod JS)</a></h3>';
 }
 
-module.exports = app;
+module.exports = app;  // eslint-disable-line no-undef
```

Comment:
What's undefined?

Reply:
Linter complains that `module` is undefined.

Reply:
Removed after setting `env` to `node`.